build/gh-pages-app.js

```diff
@@ -131,8 +140,27 @@ class GhPagesApp {
     return result;
   }
 
-  _compileHtml() {
-    let htmlSrc = this._resolveSourcesList([this.opts.html])[0];
+  /**
+   * @param {string} input
+   * @return {Promise<string>}
+   */
+  async _rollupSource(input) {
+    const plugins = [
```

Comment:
do you think it will get to the point where we want to be sharing these plugins?

Reply:
Probably not, it's at most three plugins that seem to be always used (resolve, commonjs and terser), and it's simple enough to just import them all when needed.